frontend/pages/hosts/ManageHostsPage/HostTableConfig.tsx

```diff
@@ -200,41 +199,11 @@ const allHostTableHeaders: IDataColumn[] = [
     Cell: (cellProps: INumberCellProps): JSX.Element => {
       const { id, percent_disk_space_available } = cellProps.row.original;
 
-      if (cellProps.cell.value === 0) {
-        return <>No data available</>;
-      }
-      return (
-        <>
-          <div
-            className="gigs_disk_space_available__cell__disk-space-wrapper"
-            data-tip
-            data-for={`disk-space__${id}`}
-          >
-            <div className="gigs_disk_space_available__cell__disk-space">
-              <div
-                className={`gigs_disk_space_available__cell__disk-space-${diskSpaceIndicator(
-                  cellProps.cell.value
-                )}`}
-                style={{
-                  width: `${100 - percent_disk_space_available}%`,
-                }}
-              />
-            </div>
-          </div>
-          <ReactTooltip
-            place="bottom"
-            type="dark"
-            effect="solid"
-            backgroundColor="#3e4771"
-            id={`disk-space__${id}`}
-            data-html
-          >
-            <span className={`tooltip__tooltip-text`}>
-              {diskSpaceTooltip(cellProps.cell.value)}
-            </span>
-          </ReactTooltip>{" "}
-          <span>{cellProps.cell.value} GB</span>
-        </>
+      return renderDiskSpaceGraph(
+        "gigs_disk_space_available__cell",
+        cellProps.cell.value,
```

Comment:
You might like to de-structure `gigs_disk_space_available` along with the other values from `cellProp.row.original`, which might be more easy to read than referencing `cellProps.cell.value`.